From f4088fee52784f3efb4a2545c265c1c923f89223 Mon Sep 17 00:00:00 2001
From: Bruce Forstall <brucefo@microsoft.com>
Date: Fri, 8 Dec 2017 16:30:26 -0800
Subject: [PATCH] Disable finalizertest for ARM

It is already disabled in issues.targets and testsFailingOutsideWindows.txt.

It has been failing on Windows ARM under MinOpts.
---
 tests/arm/Tests.lst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/arm/Tests.lst b/tests/arm/Tests.lst
index 3a8f97f..48df9bd 100644
--- a/tests/arm/Tests.lst
+++ b/tests/arm/Tests.lst
@@ -22625,7 +22625,7 @@ RelativePath=GC\LargeMemory\Allocation\finalizertest\finalizertest.cmd
 WorkingDir=GC\LargeMemory\Allocation\finalizertest
 Expected=0
 MaxAllowedDurationSeconds=600
-Categories=EXPECTED_PASS;Pri1
+Categories=EXPECTED_FAIL;3392;Pri1
 HostStyle=0
 
 [_il_reltest1.cmd_2842]
-- 
2.7.4

